### ISSUE TITLE
Http headers appear in RED color 

### ISSUE THREAD

**Issue by cmoulliard** [jakubroztocil/httpie#735]
### Issue 

Is it possible to change the colors of the headers as they appear in red when I run a httpie request

```
 ls -la /usr/local/bin/http
 ../Cellar/httpie/1.0.2/bin/http
http --version
1.0.2
http://$route_address/api/client/2
HTTP/1.1 200 
Cache-control: private
Content-Type: application/json;charset=UTF-8
Date: Thu, 06 Dec 2018 18:26:34 GMT
Set-Cookie: ed92a83990a1834ab681cc2334241cd1=547aa9cfaa874ed4b0b40c806379c928; path=/; HttpOnly
Transfer-Encoding: chunked

{
    "id": 2,
    "name": "Apple"
}
```

![screenshot 2018-12-06 19 28 17](https://user-images.githubusercontent.com/463790/49604039-228a6400-f98d-11e8-9226-c7b619d5cae6.png)

### Debug info

```
HTTPie 1.0.2
Requests 2.20.1
Pygments 2.3.0
Python 3.7.1 (default, Nov 28 2018, 11:51:47)
[Clang 10.0.0 (clang-1000.11.45.5)]
/usr/local/Cellar/httpie/1.0.2/libexec/bin/python3.7
Darwin 18.2.0

<Environment {
    "colors": 256,
    "config": {
        "__meta__": {
            "about": "HTTPie configuration file",
            "help": "https://httpie.org/doc#config",
            "httpie": "1.0.2"
        },
        "default_options": "[]"
    },
    "config_dir": "/Users/dabou/.httpie",
    "is_windows": false,
    "stderr": "<_io.TextIOWrapper name='<stderr>' mode='w' encoding='UTF-8'>",
    "stderr_isatty": true,
    "stdin": "<_io.TextIOWrapper name='<stdin>' mode='r' encoding='UTF-8'>",
    "stdin_encoding": "UTF-8",
    "stdin_isatty": true,
    "stdout": "<_io.TextIOWrapper name='<stdout>' mode='w' encoding='UTF-8'>",
    "stdout_encoding": "UTF-8",
    "stdout_isatty": true
}>
```

**Comment by jakubroztocil** [comment]
@cmoulliard this looks strange. Starting with version 1.0.0 httpie uses the native terminal colors (`--style=auto`). So I'd try tweaking the terminal colors. If that is not an option or if it doesn't help, then you can change it from the default scheme via the config file, for example:

```bash
$ cat ~/.httpie/config.json
```

```json
{
    "__meta__": {
        "about": "HTTPie configuration file",
        "help": "https://httpie.org/doc#config",
        "httpie": "1.0.0"
    },
    "default_options": [
        "--style=fruity"
    ]
}
```

**Comment by maelvalais** [comment]
I happen to have the exact same issue. I use iTerm2 (3.2.7beta3) + Zsh (and oh-my-zsh +agnoster). Same issue with macOS's default Terminal.app.

<img width="773" alt="capture d ecran 2019-01-24 a 14 20 50" src="https://user-images.githubusercontent.com/2195781/51680878-45c3ae00-1fe3-11e9-8536-4af8d4febd8c.png">

**Comment by Guillaume-Mayer** [comment]
Same issue here.
I think it is because the "OK" is missing in the status line (ie "200" instead of "200 OK").
Somer servers don't return it, this is the default in Tomcat for example.
[RFC 7230 states that clients should ignore it.](https://tools.ietf.org/html/rfc7230#page-22)

**Comment by Josholith** [comment]
As HTTPie uses Pygments to do syntax highlighting, we had to make a fix for RFC 7230 adherence upstream in Pygments.

This was fixed in Pygments PR #1432 https://github.com/pygments/pygments/pull/1432, and was released in Pygments 2.7.0 (September 12, 2020)  https://pygments.org/docs/changelog/#version-2-7-0

Using HTTPie with Pygments 2.7.0 or later should resolve this issue.
